src/formatting.rs

```diff
@@ -52,7 +58,21 @@ impl FormatTemplate {
         false
     }
 
-    pub fn from_string(s: &str) -> Result<Self> {
+    // TODO: Remove?
```

Comment:
Only `xrandr` uses it, right?

Reply:
And `pacman` but that might change. I can replave both of them